dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -1797,7 +1797,7 @@ info:
     url: https://github.com/dockstore/dockstore/blob/develop/LICENSE
   termsOfService: TBD
   title: Dockstore API
-  version: 1.10.0-alpha.3-SNAPSHOT
+  version: 1.11.0-alpha.0-SNAPSHOT
```

Comment:
Seems wrong, should be 1.10...

Reply:
weird, re-running mvn does not change this